kerastuner/engine/hyperparameters.py

```diff
@@ -483,6 +483,21 @@ def conditional_scope(self, parent_name, parent_values):
         finally:
             self._scopes.pop()
 
+    def is_active(self, name):
+        """Checks whether a conditional hyperparameter is active.
+
+        A hyperparameter is only considered active for the current trial if
+        its parent's conditions are met. For more information, see
+        `HyperParameters.conditional_scope`.
+
+        # Arguments:
+          name: The name of the hyperparameter.
+
+        # Returns:
+          Whether the hyperparameter is active for the current trial.
+        """
+        return self._get(name, error_on_inactive=False) is not None
+
     def _conditions_are_active(self, scopes=None):
```

Comment:
Is there a way to make this work with loops ie. multiple layers? It seems like to apply this to layers we need to write out each layer and have an conditional hyper param for each? Is there a way for the program to automatically detect if a param is active or not without having to supply the conditional HP?